src/app.ts

```diff
@@ -1,12 +1,13 @@
 import express, { Request, Response } from 'express';
 import { clientError, serverError } from './middlewares';
+import therapists from './routes';
 
 const app = express();
 
 app.get('/', (req: Request, res: Response) => {
   res.send('Hello World!');
 });
-
+app.use(therapists);
```

Comment:
Here we need to use our main Router, at `/router/index.ts` please import all subrouter and use it there